chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -20,8 +20,8 @@ BOOTPROTO=none
 <% else -%>
 BOOTPROTO=static
   <% unless @nic.kind_of?(Nic::Vlan) or @nic.kind_of?(Nic::Bond) -%>
-    <% unless @enable_tx_offloading -%>
-ETHTOOL_OPTIONS="-K iface tx off"
+    <% if @ethtool_options -%>
```

Comment:
You can't use a string as a boolean the way you expect it:

```
puts "ok" if ""
(irb):8: warning: string literal in condition
ok
```

You need to check if the string is empty or not.